library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss

```diff
@@ -490,7 +493,11 @@
       color: var(--sakai-text-color-2);
   }
 
-  .lastmodified, .groupsforthispage { font-size: smaller; text-align: right; color: #555;padding:.5em 0}
+  .lastmodified, .groupsforthispage { 
+      font-size: smaller; 
+      text-align: right; 
+      padding:.5em 0;
+   }
   
   .bold-text { font-weight: bold; color:#000 }
```

Comment:
@ottenhoff Perhaps we can add `fw-bold` class to view.vm line 91. I don't see this style used anywhere else. then we can remove it. I can do a pr in Matt's branch for this or you can just edit it , if you have edit privileges.
```suggestion
```